.circleci/config.yml

```diff
@@ -0,0 +1,21 @@
+version: '2.1'
+orbs:
+  shellcheck: circleci/shellcheck@2.2.4
+  slack: circleci/slack@4.2.1
+workflows:
+  server-scripts-ci:
+    jobs:
+      - shellcheck/check:
+          dir: .
+          post-steps:
+            - slack-notify-on-failure
```

Comment:
where is this going to notify to? 
I _think_ there is a context that we should wrap this with to be able to shepherd this to our server-alerts channel.

Reply:
Fair point, added a context